packages/blocks/src/_common/utils/render-linked-doc.ts

```diff
@@ -346,79 +354,151 @@ async function renderImageAbstract(
   card.isBannerEmpty = false;
 }
 
-function moveBlocksToLinkedDoc(
-  doc: Doc,
-  linkedDoc: Doc,
+export function addBlocksToNote(
```

Comment:
Not sure if the name should be `addBlocksToDoc` ?

Reply:
Make sense. I have changed the function name, thanks for your comment!